services/provider/server/consumer.go

```diff
@@ -211,24 +218,44 @@ func (c *ocsConsumerManager) UpdateStatusLastHeatbeat(ctx context.Context, id st
 	}
 	c.mutex.RUnlock()
 
-	patchInfo := struct {
-		Op    string      `json:"op"`
-		Path  string      `json:"path"`
-		Value interface{} `json:"value"`
-	}{
-		Op:    "replace",
-		Path:  "/status/lastHeartbeat",
-		Value: metav1.Now(),
+	patchInfo := []statusPatch{
```

Comment:
Shouldn't it be a good idea to do a status update instead of the patch as we are updating 3 fields now and maybe more in the future?

Reply:
got it, it makes sense now why the initial looked unusual.

Reply:
moved from `Patch` to `Update` as suggested